### DISCUSSION TITLE
Zk Community Chat

### DISCUSSION THREAD

**Discussion by mbfraga** [zk-org/zk#454] [General]
Hi, been using zk with neovim lately. I have taken a lot of inspiration from the bits and pieces I can find online and my own notes to make a robust note-taking environment. However, I still find it hard to find answers and bounce ideas off of people.

Is there any existing chat-based community or will to form one? There are a lot of gaps in the documentation both with zk specifically and a wider workflow involving neovim plugins and external tools.

If so, I'd love to participate in it, and potentially use it as a springboard to contribute more to the actual project.

**Comment by tjex** [comment]
Hello. So far we only have the discussions here on GitHub.

I like the idea of having a community hub, more so now that I recently found out about [Zulip](https://zulip.com/) - I can't stand slack or discord. I do like Matrix and IRC, but there is a large barrier to entry there, which creates quite a bit of friction for joining.

I'm interested to see how much interest this gets over the coming months and then set something up if there is interest. I say the coming months because I'm juggling quite a bit at the moment (masters thesis and looking for a student developer job), and don't want to add something else to my plate at the cost of being able to keep up with bugs / PRs / project quality / reliability, properly. That's priority number 1 for me.

🚨 _So if other people see this and are even tentatively interested in having a better space for community discussion / interaction than GitHub discussions, react with some kind of emoji or comment!_

In the meantime @mbfraga, #453 is in progress. It's a proper docs site for zk. If you like, you could branch off that PR and fill in the documentation holes you've found. Once #453 is merged, then you can make a PR to merge your docs changes. `zk-nvim` docs are just going to stay on GitHub for now.

**Comment by mbfraga** [comment]
Yeah, it seems Today projects are super fragmented in terms of where to find communities. While I have a strong opinion (always preferring open source solutions), any community is better than none at this point. I've admitted defeat on choice of chat protocol haha.

Good luck with that! It does sound like a lot--no worries.

I did see that and it was a godsend! Glad that it's merged :) I'll take a look!
